components/faqs/css-transition-group.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import Code from '../Code'
+
+const sampleCssTransitionGroup = (`
+import {CSSTransitionGroup} from 'react-transition-group'
+
+// Define constants for all our class names and timeouts.
+// This is optional, but recommended.
+const enter = 'example-enter'
+const enterActive = 'example-enter-active'
+const leave = 'example-leave'
+const leaveActive = 'example-leave-active'
+const enterTimeout = 500
+const leaveTimeout = 500
+
+// Set the props that CSSTransitionGroup expects, including mapping its
+// desired class names to the ones we've defined.
+const ExampleTransitionGroup = styled(CSSTransitionGroup).attrs({
+  transitionName: { enter, enterActive, leave, leaveActive },
+  transitionEnterTimeout: enterTimeout,
+  transitionLeaveTimeout: leaveTimeout,
+})\`
+  .\${enter} {
+    transform: translateY(100%);
+  }
+
+  .\${enterActive} {
+    transform: translateY(0);
+    transition: transform \${enterTimeout}ms ease-in;
+  }
+
+  .\${leave} {
+    transform: translateY(0);
+  }
+
+  .\${leaveActive} {
+    transform: translateY(100%);
+    transition: transform \${leaveTimeout}ms ease-in;
+  }
+\`
+
+render(
+  <ExampleTransitionGroup>
+    <div key={1}>Thing 1</div>
+    <div key={2}>Thing 2</div>
+    <div key={3}>Thing 3</div>
+  </ExampleTransitionGroup>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Integrating with CSSTransitionGroup">
```

Comment:
So, the other FAQ items have question-titles (eg. "Can I refer to other components?"). I'm breaking with convention here :x

The reason is that the menu items aren't in question-format, and the menu item needs to match the section header for the hash anchors to work. I've opened an issue (https://github.com/styled-components/styled-components-website/issues/53), but in the meantime I've opted to make the link work, over following the heading convention.